source/Octopus.Cli/CliProgram.cs

```diff
@@ -22,11 +22,10 @@ public class CliProgram
     {
         public int Execute(string[] args)
         {
-#if !HTTP_CLIENT_SUPPORTS_SSL_OPTIONS
```

Comment:
We hadn't defined this in this project, so this was always included (up until we added it in this pr)